metastats-domain/src/main/groovy/life/qbic/metastats/filter/FilterExperimentData.groovy

```diff
@@ -39,6 +40,7 @@ class FilterExperimentData implements FilterExperimentDataInput {
         this.messageOutputPort = messageOutputPort
         this.mapper = mapper
         this.validator = validator
+        validFilenames = false
```

Comment:
NIT: should this not be `this.validFilenames`?

Reply:
This seems to be a leftover, I deleted this variable